src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { render, waitFor } from 'utils/test-utils'
+
+import Header from './Header'
+import { skeletonId } from './constants'
+import { user } from './mock'
+
+describe('WHEN I visit the Job description page', () => {
```

Comment:
You forgot your GIVEN

Reply:
You can omit it :)

https://www.guru99.com/gherkin-test-cucumber.html

> However, you can omit writing user interactions in Given steps if Given in the "Precondition" step.